app/controllers/community_resource_controller.rb

```diff
@@ -0,0 +1,27 @@
+class CommunityResourceController < ApplicationController
+  def index
+    @community_resources = CommunityResource.all
+  end
+
+  def new
+    @community_resource = CommunityResource.new
+  end
+
+  def create
+    @community_resource = CommunityResource.new(community_resource_params)
+    @community_resource.save
```

Comment:
let's keep in mind whether we'll be adding any validations to this model that we'll need to set an if/else statement with errors.